src/Satooshi/Bundle/CoverallsV1Bundle/Api/Jobs.php

```diff
@@ -152,7 +152,7 @@ protected function upload($url, $path, $filename)
             'multipart' => [
                 [
                     'name' => $filename,
-                    'contents' => fopen($path, 'r'),
+                    'contents' => file_get_contents($path),
```

Comment:
hm... during real execution, with `fopen` all works, with `file_get_contents` I got
> Client error occurred. status: 422 Unprocessable Entity
JSON file not found or failed to parse.

:/